common/components/ProfilePage/index.tsx

```diff
@@ -97,9 +97,11 @@ const ProfilePage: React.FC = () => {
           </div>
 
           <div className={s.Info}>
-            <Card size="small" title="Роль">
-              {router.query.id ? user?.role : <RoleSwitcher />}
-            </Card>
+            {user?.role === Roles.ADMIN && (
+              <Card size="small" title="Роль">
+                {router.query.id ? user?.role : <RoleSwitcher />}
+              </Card>
+            )}
```

Comment:
user?.role === Roles.ADMIN u can separated in const
==> const isAdmin = user?.role === Roles.ADMIN==>
{isAdmin && (